packages/lesswrong/lib/ultraFeed/UltraFeedRepo.ts

```diff
@@ -0,0 +1,413 @@
+import { FeedItemSourceType, FeedSpotlight, PreDisplayFeedComment, PreDisplayFeedCommentThread, FeedFullPost, FeedPostMetaInfo, FeedCommentsThread, FeedCommentFromDb } from '../../components/ultraFeed/ultraFeedTypes';
```

Comment:
This file should be in the same (server) folder as the other repos; it's 100% server-side code.